index.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-module.exports = require('./lib/dependency-checker');
+var Reporter = require('./lib/reporter');
+var DependencyChecker = require('./lib/dependency-checker');
+
+module.exports = {
+  name: 'ember-cli-dependency-checker',
+
+  included: function(app) {
```

Comment:
We want to check dependencies on `init`, not in the `included` hook since only build commands invoke `included`. See @rwjblue 's comment on https://github.com/quaertym/ember-cli-dependency-checker/pull/23.